DependencyInjection/EzSystemsEzSupportToolsExtension.php

```diff
@@ -23,5 +36,41 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('services.yml');
         $loader->load('default_settings.yml');
+
+        $configuration = $this->getConfiguration($configs, $container);
+        $config = $this->processConfiguration($configuration, $configs);
+
+        if (isset($config['system_info']) && $config['system_info']['powered_by']['enabled']) {
+            $container->setParameter(
+                'ezplatform_support_tools.system_info.powered_by.name',
+                $this->getPoweredByName(
+                    $container,
+                    $config['system_info']['powered_by']['release']
+                )
+            );
+        }
+    }
+
+    private function getPoweredByName(ContainerBuilder $container, ?string $release): string
+    {
+        // Autodetect product name if configured name is null (default)
+        $vendor = $container->getParameter('kernel.root_dir') . '/../vendor/';
```

Comment:
I always wonder what's gonna happen to our Product if someone sets https://getcomposer.org/doc/06-config.md#vendor-dir :)
Maybe this entire logic should happen in a Compiler pass when loaded bundles are available (AFAIK here they're not set yet)?

Reply:
Are we overthinking it here? we have plenty of code around including autoloader from vendor folder ;)

Reply:
> Are we overthinking it here? we have plenty of code around including autoloader from vendor folder ;)

:+1: True. Because indeed it would require more work, we can keep it as is for now and handle it globally when needed/possible. It's quite minor ATM.